includes/class-sensei-grading-user-quiz.php

```diff
@@ -155,10 +155,11 @@ public function display() {
 						// Get uploaded file
 						if ( $user_answer_content ) {
 							$attachment_id    = $user_answer_content;
-							$answer_media_url = $answer_media_filename = '';
 							if ( 0 < intval( $attachment_id ) ) {
 								$answer_media_url      = wp_get_attachment_url( $attachment_id );
-								$answer_media_filename = basename( $answer_media_url );
+								$filename_raw          = basename( $answer_media_url );
+								$answer_media_filename = preg_match( '/^[a-f0-9]{32}_/', $filename_raw ) ? substr( $filename_raw, 33 ) : $filename_raw;
```

Comment:
I wonder about putting the `32` in a variable so that the naming would make it clear what its value represented. It wasn't obvious when I first saw it. Not sure if it could be used in the `preg_match` or not. If not, then setting it to `33` would make more sense instead of having to add 1.

Reply:
Not sure I understand, can you explain a bit? So what we do is that we prepend a filename with a string `<md5 hash>_`. For example the file with a name `test.txt` would turn to something like `0800fc577294c34e0b28ad2839435945_test.txt`. Now we don't want to display the in the frontend or

Reply:
Yep the login in that is not very clear. I refactored it to a common method which will make it more readable ea3d8b693b24b33a1d71ffd0e9894245bef3c47d